app/app.go

```diff
@@ -103,6 +103,10 @@ func (a *App) InitServer() {
 		if a.Srv().runjobs {
 			a.Srv().Go(func() {
 				runLicenseExpirationCheckJob(a)
+
+				if *a.Config().ServiceSettings.EnableAWSMetering {
```

Comment:
Should this be exposed as a config settings, shouldn't be enough an environment variable? If this is a config, what happens if the user disable it?

What happens if I change that setting through the API in runtime (Should we monitor config updates?)

Reply:
I don't feel super strong about it, my concern is to expose something to the user that is not needed to be exposed to the user, that settings is going to be shown in the `config/config.json` file. Not a big deal anyway.

We are not going to expose that in the admin console anyway, so I think we can keep as is now and change it later if we see any drawback.